### DISCUSSION TITLE
Guidelines

### DISCUSSION THREAD

**Discussion by detronetdip** [detronetdip/tello-server#3] [Announcements]
Please follow the below mentioned guidelines for contributing to this project.

# Guidelines

- Fork the project to your own account and make changes there.
- Set `Upstream` and `Origin` to your local repo for better DX.
- Submit your changes only via pull requests to the dev branch.
- Provide detailed explanation during submitting the pull request.
- You should not commit to the master branch directly.
- Always create a separate branch for new functionality.
- Code should be properly formatted and linted before any commit.
- Please write meaningful commit messages.
- Avoid long description in commit messages, it should be short and crisp.
- Please follow DRY (Don't Repeat Yourself) concept.
- Only commit related changes together.
- Do not merge any pull request to this repository by yourself.
- You should follow [GitHub Community Guidelines](https://docs.github.com/en/site-policy/github-terms/github-community-guidelines)

If you have any question feel free to comment in this thread.

**Comment by detronetdip** [comment]
@parijat965, @Sinchan26 and @WiZARD2121  please have a look at this.

**Comment by parijat965** [comment]
@detronetdip already checked

**Comment by Sinchan26** [comment]
Checked
